src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -3050,6 +3206,24 @@ type QueryType {
         """
         startTransitTripId: String @deprecated(reason: "Not implemented in OTP2")
     ): Plan @async
+
+    planConnection(
+        searchWindow: SearchWindowOptions
+        origin: PlanLocation
+        destination: PlanLocation
+        preferences: PlanPreferences
+        debugSettings: PlanDebugSettings
+        locale: String
+        before: String
+        after: String
+        first: Int
+        last: Int
+    ): PlanConnection @async
+}
+
+input RealtimePreferences {
+    omitCanceled: Boolean
+    ignoreRealtimeUpdates: Boolean
```

Comment:
I did something like this in https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/8372e112566973d9a99dac64e0645d7251e81464 . Not sure if the naming is better or worse, thought I should not include the REALTIME in the enum values as its already in the name of the enum. I'm not sure if we should still have a wrapper type for the realtime preferences or not. Could we, for example, imagine having some parameter that ignore delays under x minutes for example at some point or some other realtime related parameters that can't really be included in the enum?

Reply:
We discussed this in the last meeting that it could make more sense to enable/disable. Another thing we discussed was planned cancellations. It would be convenient if those could be enabled/disabled from the same field but planned cancellations are not really realtime information. Therefore, should we figure out some other umbrella term for these settings? I drafted a structure like this but the use of the term `Realtime` doesn't fully make sense here:

```
type RealtimePreferences {
    enabled: [RealtimeOption!]
}

enum RealtimeOption {
    REALTIME_ESTIMATES
    REALTIME_CANCELLATIONS
    PLANNED_CANCELLATIONS
}
```

Enabling everything would be done through not defining the parameter.

Reply:
I have now made it an input type with boolean flags again because this enum stuff turned out to be slightly complex and confusing